### PR TITLE
#494 fix. AUCTION_END event was getting emitted in multiple scenarios

### PR DESCRIPTION
**Pull request by jaiminpanchal27**
#494 . AUCTION_END event was getting emitted in multiple scenarios

### CODE REVIEW

**Comment by mercuryyy** [comment]
I tried using your build fix i get this:

INFO: Prebid.js v0.12.0-pre loaded 
pb12v2.js:1 INFO: Invoking pbjs.setPriceGranularity : params :  ["medium"]
pb12v2.js:1 INFO: Invoking pbjs.addAdUnits : params :  [Array[3]]
pb12v2.js:1 INFO: Invoking pbjs.requestBids : params :  [Object]
pb12v2.js:1 MESSAGE: Emitting event for: auctionInit
pb12v2.js:1 MESSAGE: CALLING BIDDER ======= rubicon
pb12v2.js:1 MESSAGE: Emitting event for: bidRequested
pb12v2.js:1 INFO: Invoking pbjs.setTargetingForGPTAsync 
pb12v2.js:1 MESSAGE: Rubicon Project bidding complete: 3008
pb12v2.js:1 MESSAGE: Emitting event for: bidAdjustment
pb12v2.js:1 MESSAGE: Emitting event for: bidResponse
pb12v2.js:1 MESSAGE: Emitting event for: bidTimeout
pb12v2.js:1 MESSAGE: Prebid auction cleared
pb12v2.js:1 MESSAGE: Emitting event for: auctionEnd
pb12v2.js:1 MESSAGE: Emitting event for: bidAdjustment
pb12v2.js:1 MESSAGE: Emitting event for: bidResponse
pb12v2.js:1 MESSAGE: Emitting event for: bidTimeout
pb12v2.js:1 MESSAGE: Emitting event for: bidAdjustment
pb12v2.js:1 MESSAGE: Emitting event for: bidResponse
pb12v2.js:1 MESSAGE: Emitting event for: bidTimeout


so it clears only once now but then keeps on rolling

File: src/bidmanager.js [hunk 1]
```diff
@@ -244,11 +244,10 @@ exports.executeCallback = function () {
 
   //execute one time callback
   if (externalOneTimeCallback) {
```
Comment (protonate) [inline src/bidmanager.js]: We need to `clearAuction` whether there is an `externalOneTimeCallback` or not.

**Comment by protonate** [comment]
Please update with latest master and resolve conflicts.

**Comment by protonate** [comment]
Looks good, thanks for updates, reviewed.

**Pull request merged**